integ/workflow/reset/routers.go

```diff
@@ -0,0 +1,168 @@
+package reset
+
+import (
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"github.com/indeedeng/iwf/gen/iwfidl"
+	"github.com/indeedeng/iwf/service"
+	"log"
+	"net/http"
+	"strconv"
+)
+
+//A workflow without calling WaitUntil api for any state. It loops through state2 5 times. Used for:
+// * Testing reset by state id and state execution id.
```

Comment:
I still need to document all the integ test like this format from @samuel27m https://github.com/indeedeng/iwf/pull/456/files

It would be nice to follow so that it's much more clear to know the workflow